### ISSUE TITLE
Ability to select out-of-vision areas

### ISSUE THREAD

**Issue by sanderd17** [openstreetmap/iD#1420]
When you're completely inside an area (like a big landuse one), and you click somewhere random between the streets, you select that landuse.

Because you can mess up the entire form with a single click (e.g. square it, or make it round), without even noticing you are doing something, newbies can do a lot of damage without knowing.

So my proposal is, from the moment you don't see the boundaries of the landuse any more, you shouldn't be able to select it (the chances the you want to edit something invisible are rather small anyway). Or maybe notify the user he's editing something big (outside his screen).

Sorry if it has been reported before, I didn't read the entire bug list.

Anyway, thanks for the great new editor.

TL;DR. a.k.a STEPS TO REPRODUCE:

Zoom into a big landuse area until you don't see the borders anymore
Click inside it
You can edit the landuse without seeing it.
This shouldn't happen
